--- origsrc/libdvdread-5.0.3/Makefile.am	2014-08-28 01:41:51.000000000 -0500
+++ src/libdvdread-5.0.3/Makefile.am	2015-05-19 15:08:18.399151800 -0500
@@ -34,6 +34,7 @@ libdvdread_la_SOURCES = \
 libdvdread_la_LIBADD = $(CSS_LIBS)
 
 libdvdread_la_LDFLAGS = -version-info $(DVDREAD_LTVERSION) \
+	-no-undefined \
 	-export-symbols-regex "(^dvdread.*|^nav.*|^ifo.*|^DVD.*|^UDF.*)"
 
 pkgincludedir = $(includedir)/dvdread
